Software-Engineering.md

```diff
@@ -1,8 +1,10 @@
 # Software Engineering Roles
 
-## Software Engineer
+## Junior Software Engineer / Software Engineer
 
-> I am a valuable, enthusiastic and net-positive member of an Octopus team.
+_We avoid distinguishing too much between the Junior Software Engineer and Software Engineer. These roles are primarily focused on learning with the aim to grow into the contribution of a Senior Software Engineer or beyond. The primary indicator inside these levels is how much help you need to be a net-positive contributor._
```

Comment:
I think we should move away from the phrase"net-positive contributor" - I've seen it used in a few places. The phrase implies people might be negative contributors, which isn't correct, even for our juniors - they all contribute something.

Could we consider instead the phrasing "dependent contributors" and "independent contributors"? So the last line would read:

> The primary indicator inside these levels is how far you are along your journey from dependent contribution to independent contribution